vue-components/tests/unit/components/TextArea.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { mount } from '@vue/test-utils';
+import TextArea from '@/components/TextArea.vue';
+import { ResizeLimit } from '@/components/ResizeLimit.ts';
+
+describe( 'TextArea.vue', () => {
+	it( 'accepts rows property', () => {
+		const wrapper = mount( TextArea, {
+			propsData: { rows: 42 },
+		} );
+
+		expect( wrapper.props().rows ).toBe( 42 );
+		expect( wrapper.find( 'textarea' ).attributes( 'rows' ) ).toBe( '42' );
+	} );
+
+	it( 'accepts resize property', () => {
+		const wrapper = mount( TextArea, {
+			propsData: { resize: ResizeLimit.Horizontal },
+		} );
+
+		expect( wrapper.props().resize ).toBe( ResizeLimit.Horizontal );
+		expect( wrapper.find( 'textarea' ).classes() ).toContain( 'wikit-TextArea__textarea--horizontal' );
+	} );
+
+	it( 'ignores invalid resize values', () => {
+		expect( () => mount( TextArea, {
+			propsData: { resize: 'nonsense' },
+		} ) ).toThrow( 'Invalid prop: custom validator check failed for prop "resize"' );
```

Comment:
I am actually of a really split mind here, since it seemed that the original WiKit hike felt like it was necessary to throw in case anything [logs an error or warning](https://github.com/wmde/wikit/blob/74d77539d9770c7104cea614b99dd5082f2fb61f/vue-components/tests/config/JestCustomEnvironment.js) during a test run.

Therefore, I had to check for throwing errors even though this behavior is not implemented by vue itself, as the former assertion we had here wasn't actually executed (the test would throw even before it gets to the assertion).

I wouldn't change this throwing test double now though, since it seems some other tests are relying on it too, so I made an executive decision to just follow suite.